asset-transfer-basic/rest-api-typescript/src/assets.router.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Contract } from 'fabric-network';
+import { StatusCodes, getReasonPhrase } from 'http-status-codes';
+import express, { Request, Response } from 'express';
+
+import { logger } from './logger';
+
+const { INTERNAL_SERVER_ERROR, OK } = StatusCodes;
+
+export const assetsRouter = express.Router();
+
+assetsRouter.get('/:assetId', async (req: Request, res: Response) => {
+  try {
```

Comment:
We've not done this is the other REST servers, but should we apply some validation or URL decoding on the asset id?

Of course it totally depends on the datamodel of `assetid`

Reply:
Probably, although I don't think there's any particular data model for asset ID in the samples

Reply:
meant to ask what the 'backing' chaincode was... all makes sense then. 
maybe just add a comment 

```
// no requirement to validate asset id first
```

Reply:
I've added a comment to the top of the file (will probably need finessing at some point!)